v4_to_v5_migration_guide.md

```diff
@@ -0,0 +1,54 @@
+### New features
+
+#### Amazon store support
+We have introduced support for using the Amazon Store. You can enable Amazon Store support by configuring the SDK using the new configure function:
+```
+Purchases.configure({ apiKey: "AMAZON_API_KEY", useAmazon: true });
+```
+
+For more information around configuration please take a look at the [Amazon Store section in our docs](https://docs.revenuecat.com/docs/amazon-platform-resources). The official [Amazon In-App Purchasing docs](https://developer.amazon.com/docs/in-app-purchasing/iap-overview.html) also contain very valuable information, specially around testing and best practices.
+
+
+#### Apple AdServices support
+New method for automatic collecting of attribution tokens on iOS and macOS using AdServices - `Purchases.enableAdServicesAttributionTokenCollection()`
+
+#### New types and cleaned up naming
+New types that wrap native types from Apple, Google and Amazon, and we cleaned up the naming of other types and methods for a more consistent experience.
```

Comment:
FFTI. Should we add what new types were added? There is a section below for renames, but not for additions...